docs/reference/manifest/supertip.md

```diff
@@ -13,8 +13,8 @@ Defines a rich tooltip (both Title and Description). It is used by both [Button]
 
 |  Element |  Required  |  Description  |
 |:-----|:-----|:-----|
-|  [Title](#title)        | Yes |   The text for the supertip.         |
-|  [Description](#description)  | Yes |  The description for the supertip.    |
+| [Title](#title) | Yes | The text for the supertip. |
+| [Description](#description) | Yes | The description for the supertip.<br>**Note**: For Outlook, only Windows and Mac clients support the **Description** element. |
```

Comment:
Since this is in the Description line of the table, you could simplify the note to "Note: (Outlook) Only Windows and Mac clients are supported."